Source/Services/Network/Services/RepositoryService.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  RepositoryService.swift
+//  GithubNotifications
+//
+//  Created by Su T. Nguyen on 20/09/2021.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import Combine
+
+// sourcery: AutoMockable
+protocol RepositoryServiceProtocol {
+
+    func getRepositories() -> AnyPublisher<[Repository], Error>
+}
+
+final class RepositoryService: RepositoryServiceProtocol {
+
+    private let networkAPI: API
+    private let userSession: UserSessionStorageProtocol
+
+    init(
+        networkAPI: API,
+        userSession: UserSessionStorageProtocol
+    ) {
+        self.networkAPI = networkAPI
+        self.userSession = userSession
+    }
+
+    func getRepositories() -> AnyPublisher<[Repository], Error> {
+//        networkAPI.performRequest(
+//            with: RepositoryRequestConfiguration.getRepositories(userSession.username ?? "")
+//        )
+
+        Future { promise in
```

Comment:
As task #19 is backend task, is there a bug that makes you need to use a mock data?